package.json

```diff
@@ -4,41 +4,18 @@
 	"private": true,
 	"scripts": {
 		"serve": "vue-cli-service serve",
-		"build": "vue-cli-service build",
-		"lint": "vue-cli-service lint"
+		"build": "vue-cli-service build"
 	},
 	"dependencies": {
 		"core-js": "^3.6.5",
-		"vue": "^2.6.11"
+		"vue": "^2.6.11",
+		"vue-focus-visible": "^2.1.0"
 	},
 	"devDependencies": {
 		"@vue/cli-plugin-babel": "~4.5.0",
-		"@vue/cli-plugin-eslint": "~4.5.0",
 		"@vue/cli-service": "~4.5.0",
-		"babel-eslint": "^10.1.0",
-		"eslint": "^6.7.2",
-		"eslint-plugin-vue": "^6.2.2",
 		"vue-template-compiler": "^2.6.11"
 	},
-	"eslintConfig": {
```

Comment:
I'm sorry Quinn, your linter is just too agressive for me.

Reply:
Fair I agree lol it's the default I don't love it thought